index.js

```diff
@@ -1,20 +1,16 @@
 "use strict";
 
 module.exports = {
```

Comment:
Not a big deal, just FYI, we don't have linting rules in libraries, so it is usually automatically formatted on save by code editors (according to local configurations), and that creates unnecessary diffs. I usually "Save without formatting", so that the previous formatting won't change